src/pcm/PcmDsd.cxx

```diff
@@ -63,7 +63,10 @@ PcmDsd::ToFloat(unsigned channels, ConstBuffer<uint8_t> src) noexcept
 			if (dsd2pcm[c] == nullptr)
 				return nullptr;
 		}
-
+	}
+	unsigned c;
+#pragma omp parallel for
```

Comment:
Did you measure how much this really speeds up?
This will generate a huge lot of cache line bouncing, so I doubt the usefulness of this change. This would rather increase overall CPU usage, and stall several CPUs, and this CPU time will not be available to other threads/processes.

Reply:
Just for caryaudio.  When playing DSD512, there is underrun all the time.  Use this, no underrun happenned.